.github/ISSUE_TEMPLATE/folders.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Translation for main folders
+about: Do you want to translate the main folders of the Wiki to your language?
+title: Translation for main folders
+---
+Hi,
+
+I have read [the Styleguide](https://wiki.openttd.org/en/Wiki/Manual%20of%20Style#folders) and want to provide translations for the main wiki folders.
+
+Please rename all folders and update links accordingly.
```

Comment:
Somewhere in this issue template is missing for what language this is requested :D